apps/molgenis-components/src/client/client.js

```diff
@@ -225,22 +224,26 @@ const fetchTableData = async (
   const resp = await axios
     .post(url, { query: tableDataQuery, variables: { filter, orderby } })
     .catch((error) => {
-      console.log(error);
       if (typeof onError === "function") {
         onError(error);
       }
     });
   return resp.data.data;
 };
 
-const request = async (url, graphql) => {
+const request = async (url, graphql, variables) => {
+ const data = { query: graphql };
+ if(variables) {
+   data.variables = variables;
+ }
   return axios
-    .post(url, { query: graphql })
+    .post(url, data)
     .then((result) => {
       return result?.data?.data;
     })
     .catch((error) => {
       throw error;
+      error
```

Comment:
the error is no longer throw , nor is it returned

Reply:
ah, that is an accident LOL, thanks